src/08-accessing.js

```diff
@@ -116,7 +136,13 @@ function getLastPhoneNumber(person) {}
  *  getFullName(person);
  *  //> "Rachel Eleanor Rojas"
  */
-function getFullName(person) {}
+function getFullName(person) {
+  let firstName = person.names.first;
+  let midName = person.names.middle;
+  let lastName = person.names.last;
+  let fullName = firstName.concat(" ", midName ," ", lastName);
```

Comment:
Nice work - I would recommend moving away from the `concat()` method and starting to work with template literals and backticks.  They're just a little smoother to read.